frontend/src/ModelOverview.tsx

```diff
@@ -9,6 +9,7 @@ import EntitiesDisplay from '../components/EntitiesDisplay'
 import { useGetSchema } from '../data/schema'
 import { ModelDoc, Version } from '../types/types'
 import { printProperty } from '../utils/propertyUtils'
+import HelpPopover from './/common/HelpPopover'
```

Comment:
This path is invalid, should probably be './common/HelpPopover'

Reply:
Sorted